ocaml-lsp-server/test/e2e/__tests__/textDocument-signatureHelp.ts

```diff
@@ -186,20 +208,22 @@ describe_opt("textDocument/completion", () => {
 
   it("can make an optional parameter active by prefix", async () => {
     openDocument(outdent`
-      let _ = Hashtbl.create ?ra
+      let create = Hashtbl.create
+
+      let _ = create ?ra
     `);
 
-    let items = await querySignatureHelp(Types.Position.create(0, 26));
+    let items = await querySignatureHelp(Types.Position.create(2, 18));
     expect(items).toMatchObject({
       signatures: [
         {
-          label: "Hashtbl.create : ?random:bool -> int -> ('a, 'b) Hashtbl.t",
+          label: "create : ?random:bool -> int -> ('a, 'b) Hashtbl.t",
```

Comment:
Why did the name change here? `Hashtbl.create` seems like a better label.

Reply:
> I see you've changed the position.

Yep, I made the `let create = Hashtbl.create` binding so those tests wouldn't read standard library documentation comments, which could change between releases.